examples/echo_bot/server.go

```diff
@@ -41,7 +41,7 @@ func main() {
 		cb, err := webhook.ParseRequest(channelSecret, req)
 		if err != nil {
 			log.Printf("Cannot parse request: %+v\n", err)
-			if err == linebot.ErrInvalidSignature {
+			if err.Error() == linebot.ErrInvalidSignature.Error() {
```

Comment:
Could you try 

//   if err == ErrNotFound { … }
if errors.Is(err, linebot.ErrInvalidSignature) {
    // something wasn't found
}